<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_Recent">
     <property name="title">
      <string>&amp;Recent</string>
     </property>
    </widget>
    <addaction name="action_New"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_All"/>
    <addaction name="action_Save_As"/>
    <addaction name="separator"/>
    <addaction name="menu_Recent"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_All"/>
   <addaction name="action_Save_As"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidgetProject">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeView" name="treeViewProject">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="animated">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icon/New_file.png</normaloff>:/Icon/New_file.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icon/open.png</normaloff>:/Icon/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icon/save.png</normaloff>:/Icon/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icon/save_all.png</normaloff>:/Icon/save_all.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save All</string>
   </property>
  </action>
  <action name="action_Save_As">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icon/save_as.png</normaloff>:/Icon/save_as.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save As</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>treeViewProject</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
